core/themes/basis/theme-settings.php

```diff
@@ -75,3 +75,18 @@
     '#markup' => '<p>' . t('This theme supports custom color palettes if the Color module is enabled on the <a href="!url">modules page</a>. Enable the Color module to customize this theme.', array('!url' => url('admin/modules'))) . '</p>',
   );
 }
+
+function basis_form_system_theme_settings_alter(&$form, &$form_state) {
+  $form['supplemental'] = array(
+    '#type'      => 'fieldset',
+    '#title'     => t('CSS Updates'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+  $form['supplemental']['basis_supplemental_css'] = [
```

Comment:
`[]` isn't optimal because Backdrop still tries to be backwards compatible. User `array()` instead - like above.
That should fix the failing tests on php 5.3.

Reply:
Thanks for this info. I'm a php hack and copied code from different sources and this was the result.